src/v2/Components/v2/ArtworkFilter/Utils/allowedFilters.ts

```diff
@@ -0,0 +1,61 @@
+export const allowedFilters = filterParams => {
+  return Object.keys(filterParams)
+    .filter(key => supportedInputArgs.includes(key))
+    .reduce((obj, key) => {
+      obj[key] = filterParams[key]
+      return obj
+    }, {})
+}
+
+// This corresponds to all the allowed inputs to the
+// filtered artworks connection. This can be used as an
+// allow-list when passing filter state from a URL.
+const supportedInputArgs = [
```

Comment:
This is right from the source! https://github.com/artsy/metaphysics/blob/b2fa2028e684027cf642730ccd9371200d4b5683/_schemaV2.graphql#L6007

There's prob a better way to include/reference this Relay type, vs. this hardcoded (and duplicated) list. Perhaps a  followup?